README.md

```diff
@@ -3,6 +3,16 @@
 Some scripts I've written for [Merlin's UdonSharp](https://github.com/Merlin-san/UdonSharp)
 
 
+#### > Boost.cs
+
+Player within range of the `referencePoint` gameobject will be thrown into direction of `targetPoint` with an assigned speed. 
```

Comment:
Capitalisation of `gameobject` is inconsistent with the rest of the readme. Change to `GameObject` or change other instances to `gameobject`.

Written like this sounds better to me:
> Players within a specified range of the `referencePoint` gameobject will be thrown towards the `targetPoint` gameobject with an assigned speed.

Reply:
Fixed with a new commit